treeherder/model/tasks.py

```diff
@@ -90,44 +90,43 @@ def publish_job_action(project, action, job_id, requester):
     )
 
 
-@task(name='publish-resultset-action')
-def publish_resultset_action(project, action, resultset_id, requester, times=1):
+@task(name='publish-push-action')
+def publish_push_action(project, action, push_id, requester, times=1):
     newrelic.agent.add_custom_parameter("project", project)
     newrelic.agent.add_custom_parameter("action", action)
-    newrelic.agent.add_custom_parameter("resultset_id", str(resultset_id))
+    newrelic.agent.add_custom_parameter("push_id", str(push_id))
     newrelic.agent.add_custom_parameter("requester", requester)
 
     publisher = pulse_connection.get_publisher()
     if not publisher:
         return
 
-    publisher.resultset_action(
+    publisher.push_action(
         version=1,
         project=project,
         action=action,
         requester=requester,
-        resultset_id=resultset_id,
+        push_id=push_id,
```

Comment:
Please include BOTH ``push_id`` and ``resultset_id`` as params here.  The external consumer of these events are expecting ``resultset_id``.  So we can must include using ``resultset_id`` as well as ``push_id`` for now, until everyone has transitioned.